apps/src/aichat/views/UserChatMessageEditor.tsx

```diff
@@ -38,7 +38,7 @@ const UserChatMessageEditor: React.FunctionComponent = () => {
           isIconOnly
           icon={{iconName: 'paper-plane'}}
           onClick={handleSubmit}
-          disabled={isWaitingForChatResponse}
+          disabled={isWaitingForChatResponse || !userMessage}
```

Comment:
Unrelated change  to disable the send message button until you have something in the window.